src/model/OutputTableMonthly.hpp

```diff
@@ -0,0 +1,134 @@
+/***********************************************************************************************************************
+*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+*  See also https://openstudio.net/license
+***********************************************************************************************************************/
+
+#ifndef MODEL_OUTPUTTABLEMONTHLY_HPP
+#define MODEL_OUTPUTTABLEMONTHLY_HPP
+
+#include <model/ModelAPI.hpp>
+#include "ModelObject.hpp"
+
+namespace openstudio {
+namespace model {
+
+  namespace detail {
+
+    class OutputTableMonthly_Impl;
+
+  }  // namespace detail
+
+  class MODEL_API MonthlyVariableGroup
+  {
+   public:
+    MonthlyVariableGroup(std::string variableOrMeterName, std::string aggregationType = "SumOrAverage");
```

Comment:
Monthly doesn't have a default value for digits because this field doesn't exist like it does for Annual.

Reply:
Edit: Digits doesn't exist at the group level. It does exist at the object level.

Reply:
Yes, it is an object one here, not per extensible group (one of the weirdness of E+...), and that one is set in the ctor

https://github.com/NREL/OpenStudio/blob/ea0e2a7b64f2f275f5f0f4d2257126d2bfecced4/src/model/OutputTableMonthly.cpp#L822-L827